woocommerce/woocommerce
# Add Inventory tab to the single variation page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40594



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper` (WooCommerce Beta Tester plugin) is required
3. Go to `/wp-admin/admin.php?page=wc-admin&path=/add-product&tab=variations` and add some attributes
4. Wait until variations get generated
5. Then publish the product and save its `productId` and one `variationId` you like
6. Then go to `/wp-admin/admin.php?page=wc-admin&path=/product/{productId}/variation/{variationId}&tab=inventory` (replace the `{productId}` and `{variationId}` with the values picked in point 5)
7. The `Inventory` tab except for the `Advance` section, everything should behave as in the parent product `Inventory` section 
<img width="733" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/a0765171-8512-4168-9f4b-d1780a79bec8">
<img width="689" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/79f36914-a71b-4817-8c10-56091b316374">

8. To check that the changes did take effect after saving the variation click `Preview` button on the top of the screen.
9. You should be redirected to the storefront screen where the variation is presented to the user 
<img width="944" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/91002a97-e361-481b-ba1c-1ccadc5af4f1">


16. From there you can check that all changes that you made are correct



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
